examples/riot-kitchen-sink/src/stories/addon-actions.stories.js

```diff
@@ -0,0 +1,19 @@
+import { tag, mount, storiesOf, compileNow } from '@storybook/riot';
+import { action } from '@storybook/addon-actions';
+import ButtonRaw from './Button.txt';
+
+compileNow(tag, ButtonRaw);
```

Comment:
Can `compileNow` get something that is not a `tag`?

Reply:
sorry that is a bug will fix that right away

Reply:
thanks for pointing that out.